### PR TITLE
Add snowflake app UI

### PR DESCRIPTION
**Pull request by mayankchhabra**
Use a custom html file to wrap the GoTTY logs in an app-esque UI.

### CODE REVIEW

File: apps/snowflake/data/index.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Tow Snowflake Proxy</title>
```
Comment (nevets963) [inline apps/snowflake/data/index.html:4]: There is a typo here. This should be 'Tor'

File: apps/snowflake/data/index.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+                </defs>
+            </svg>                                                  
+            <div class="app-details">
+                <span class="app-status success">&#9679; Running</span>
+                <h1 class="app-name">Tor Snowflake Proxy</h1>
+            </div>
+        </div>
+        <p class="text-muted">
+            Thank you for installing Tor Snowflake Proxy on your Umbrel to help defeat Tor censorship around the world.
```
Comment (nevets963) [inline apps/snowflake/data/index.html:113]: Isn't this helping to defeat censorship in general? Or is it only 'Tor censorship' ?

e.g. Thank you for installing the Tor Snowflake Proxy on your Umbrel to help defeat censorship around the world.

File: apps/snowflake/data/index.html (hunk 1 above)
Comment (mayankchhabra) [reply apps/snowflake/data/index.html:4]: Oops, good catch!

File: apps/snowflake/data/index.html (hunk 2 above)
Comment (mayankchhabra) [reply apps/snowflake/data/index.html:113]: Snowflake specifically is built to help people access the Tor network in countries where Tor is blocked. So while Tor helps defeat _censorship_, Snowflake helps defeat _Tor censorship_. Maybe @lukechilds can confirm?

**Pull request merged**